### PR TITLE
lib/config: Don't migrate non-HTTPS-URL discovery servers to new path

### PR DESCRIPTION
**Pull request by calmh**
### Purpose

Don't screw up unknown URLs in the migration (i.e. `default-v4` -> `default-v4v2/`). Don't panic.

### Testing

Tested manually on a v13 config...

### CODE REVIEW

File: lib/config/config.go [hunk 1]
```diff
@@ -307,12 +308,13 @@ func convertV13V14(cfg *Configuration) {
 
 	var newAddrs []string
 	for _, addr := range cfg.Options.GlobalAnnServers {
-		if addr != "default" {
-			uri, err := url.Parse(addr)
-			if err != nil {
-				panic(err)
-			}
-			uri.Path += "v2/"
+		uri, err := url.Parse(addr)
```
Comment (calmh) [inline lib/config/config.go]: "default" and so on are valid URLs with an empty scheme and will slip through this untouched.

**Comment by AudriusButkevicius** [comment]
I guess I'd rather bump config version and release 13.1?

**Comment by calmh** [comment]
Why increment config? To fix it back you mean? I don't think we have that many users with `default-v4` or `default-v6` in their configs...

**Comment by calmh** [comment]
Well, I guess it doesn't hurt to do so anyway just to capture those two cases.

**Comment by AudriusButkevicius** [comment]
Ok, but still lets do a 0.13.1 so this wouldn't be contagious?

@st-review merge

**Comment by calmh** [comment]
I'd rather wait a little longer to see what else pops up so we don't have to release a .2 in fifteen minutes. We can't release much more before we need to start removing releases or we'll leave v0.12.23 behind :(

**Comment by calmh** [comment]
Hang on and I'll add a v15 migration

**Comment by AudriusButkevicius** [comment]
I guess we can remove 0.13.0 once 0.13.1 is out?

**Comment by calmh** [comment]
There. Fixed in the original, and an additional to undo the damage.

**Comment by calmh** [comment]
Yeah, we could. It's a bit ugly though. I think we can wait a day and collect a few more fixes into .1.

**Comment by AudriusButkevicius** [comment]
@st-review merge

**Comment by st-review** [comment]
:ok_hand: Merged as 922e1407c23c20a49dbee03212ae38e1c8d5a2ad. Thanks, @calmh!

**Pull request closed without merging**